packages/kotti-ui/source/kotti-table/table/hooks.ts

```diff
@@ -535,6 +543,83 @@ export const useKottiTable = <
 							}),
 						]
 					: []),
+				...(params.value.isSelectable === 'single'
+					? [
+							columnHelper.display({
+								cell: ({ row }: CellContext<ROW, unknown>) =>
+									h(
+										'div',
+										{
+											domProps: {
+												ariaDisabled: !row.getCanSelect(),
+											},
+											on: {
+												click: (event: MouseEvent) => {
+													event.stopPropagation()
+													event.preventDefault()
+
+													const isSelected = !row.getIsSelected()
+													table.value.toggleAllRowsSelected(false)
```

Comment:
Should probably check if this row is disabled first. During FT, clicking on a disabled row cleared others row selection.